crates/voicevox_core_python_api/src/lib.rs

```diff
@@ -457,6 +537,45 @@ where
     )
 }
 
+fn to_rust_uuid(ob: &PyAny) -> PyResult<Uuid> {
+    let uuid = ob.getattr("hex")?.extract::<String>()?;
+    uuid.parse().into_py_result()
+}
+fn to_py_uuid(py: Python, uuid: Uuid) -> PyResult<PyObject> {
+    let uuid = uuid.hyphenated().to_string();
+    let uuid = py.import("uuid")?.call_method1("UUID", (uuid,))?;
+    Ok(uuid.to_object(py))
+}
+fn to_rust_user_dict_word(ob: &PyAny) -> PyResult<voicevox_core::UserDictWord> {
+    voicevox_core::UserDictWord::new(
+        ob.getattr("surface")?.extract()?,
+        ob.getattr("pronunciation")?.extract()?,
+        ob.getattr("accent_type")?.extract()?,
+        to_rust_word_type(ob.getattr("word_type")?.extract()?)?,
+        ob.getattr("priority")?.extract()?,
+    )
+    .into_py_result()
+}
+fn to_py_user_dict_word<'py>(
+    py: Python<'py>,
+    word: &voicevox_core::UserDictWord,
+) -> PyResult<&'py PyAny> {
+    let class = py
+        .import("voicevox_core")?
+        .getattr("UserDictWord")?
+        .downcast()?;
+    to_pydantic_dataclass(word, class)
+}
+fn to_rust_word_type(word_type: &PyAny) -> PyResult<UserDictWordType> {
+    let name = word_type
+        .getattr("name")
+        .unwrap()
+        .extract::<String>()
+        .unwrap();
```

Comment:
<https://github.com/VOICEVOX/voicevox_core/pull/538#discussion_r1265759581>の方針に従うと:

```suggestion
    let name = word_type.getattr("name")?.extract::<String>()?;
```